### PR TITLE
feat: make notification avatar clickable if it contains an action

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

https://app.clickup.com/t/gb02n7

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: resources/views/navbar-notifications.blade.php [hunk 1]
```diff
@@ -25,7 +28,7 @@
                             </span>
 
                             <div class="flex flex-row space-x-4">
-                                @isset($notification->data['action'])
+                                @isset($notification->data['action']['url'])
```
Comment (ItsANameToo) [inline resources/views/navbar-notifications.blade.php]: won't this crash if there is no `data['action']` property

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/views/navbar-notifications.blade.php (hunk 1 above)
Comment (Highjhacker) [reply resources/views/navbar-notifications.blade.php]: no

**Comment by ItsANameToo** [comment]
@Highjhacker this will make the avatar link to the action, not to the user or project that it shows? 🤔

**Comment by Highjhacker** [comment]
@ItsANameToo Indeed, my bad, will check on that again

File: resources/views/notification-icon.blade.php [hunk 2]
```diff
@@ -9,19 +9,22 @@
...
-        @if($media)
-            {{ $media->img('', ['class' => 'absolute object-cover w-full h-full rounded-md']) }}
-        @elseif($identifier)
-            <x-ark-avatar :identifier="$identifier" class="absolute object-cover w-full h-full rounded-md" />
-        @elseif($defaultLogo)
-            <img class="object-cover rounded-md" src="{{ $defaultLogo }}" />
-        @else
-            <div class="border border-theme-secondary-200 w-11 h-11"></div>
-        @endif
+<a @if($hasRoute) href="{{ $notification->route() }}" @endif>
```
Comment (ItsANameToo) [inline resources/views/notification-icon.blade.php]: this will result in an empty `a` tag if there is no route set :susp:

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**